dists/install.sh

```diff
@@ -131,22 +127,19 @@ main() {
   url="https://github.com/Scalingo/cli/releases/download/${version}/${archive_name}"
 
   status "Downloading Scalingo client...  "
-  curl --silent --fail --location --output ${tmpdir}/${archive_name} ${url}
-  if [ ! -f ${tmpdir}/${archive_name} ]; then
+  curl --silent --fail --location --output "${tmpdir}/${archive_name}" "$url"
+  if [ ! -f "${tmpdir}/${archive_name}" ]; then
     echo ""
     error "Fail to download the CLI archive\n"
     exit 1
   fi
   echo "DONE"
 
-  check_command_exists sha256sum
-  check_command_exists wget
-
   status "Verifying the checksum...  "
   checksums_url="https://github.com/Scalingo/cli/releases/download/${version}/checksums.txt"
   # Use cut's short parameter to be compatible with MacOS: https://ss64.com/osx/cut.html
-  checksum_computed=$(sha256sum ${tmpdir}/${archive_name} | cut -d" " -f1)
-  checksum_expected=$(wget --quiet --output-document - $checksums_url | grep $archive_name | cut -d" " -f1)
+  checksum_computed=$(sha256sum "${tmpdir}/${archive_name}" | cut -d" " -f1)
+  checksum_expected=$(curl --silent --location "$checksums_url" | grep "$archive_name" | cut -d" " -f1)
```

Comment:
This is the actual fix

Reply:
question: I have the same question as the one raised above. From what I understand of the issue, the problem is not the usage of `wget` that is here for years according to the Git history. The problem is the call to `check_command_exists`.

Reply:
Yes `check_command_exists` blocked user that didn't have `wget` installed.
But in the same way as this command failed due to missing binary.

I found it more consistent to use curl as we are already asking to use for the download https://doc.scalingo.com/platform/cli/start